### DISCUSSION TITLE
Menü mit Maustaste öffnen global deaktivieren?

### DISCUSSION THREAD

**Discussion by X-TheUser-X** [Courseplay/Courseplay_FS22#2931] [Q&A] [answered]
Hallo, 

ist es möglich, das Öffnen des CP-Fensters mit der Maustaste irgendwo global zu deaktivieren oder geht es nur über den Weg, das bei jedem Fahrzeug einzeln nach dem Kauf zu machen? 

Danke.

**Comment by Tensuko** [comment, marked as answer]
Einfach die Tastenbelegung rausnehmen.
